data/sql/updates/pending_db_world/rev_1648344440179668500.sql

```diff
@@ -0,0 +1,206 @@
+
+-- Children Position Adjustment
+UPDATE creature SET `wander_distance` = 8, `MovementType` = 1 WHERE `guid` IN (86156,86157,86158,86159,86354);
+UPDATE creature SET `position_x` = -9511.99, `position_y` = -1272.53, `position_z` = 43.669, `orientation` = 5.56961 WHERE `guid` = 86354;  -- Eric
+UPDATE creature SET `position_x` = -9507.03, `position_y` = -1275.85, `position_z` = 44.1562, `orientation` = 5.16513 WHERE `guid` = 86156; -- Jay
+UPDATE creature SET `position_x` = -9511.31, `position_y` = -1285.64, `position_z` = 44.1448, `orientation` = 3.93122 WHERE `guid` = 86159;  -- Kevin
+
+-- ------------------------------------Nocturnal (Pyrewood Village) Event----------------------------
+
+-- Nocturnal Only Creatures
+DELETE FROM `game_event_creature` WHERE `eventEntry` = 25 and `guid` in (76265,86173,86174,86831,152333,152334,152335,152336,152337,152338,152339,97406,97407,97408,97431);
+INSERT INTO `game_event_creature` (`eventEntry`,`guid`) VALUES
+(25,76265), -- Ultham Ironhorn Ram Trainer
+(25,86173), -- Rams - Brown Ram in Corral
+(25,86174), -- Rams - Gray Ram in Corral
+(25,86831), -- Rotten Ghouls replace Fleshrippers/Smugglers in Westfall
+(25,152333),
+(25,152334),
+(25,152335),
+(25,152336),
+(25,152337),
+(25,152338),
+(25,152339),
+(25,97406), -- Rams in Corral
+(25,97407), -- Swift Rams in Corral
+(25,97408),
+(25,97431);
+
+DELETE FROM `game_event_creature` WHERE `eventEntry` = -25 and `guid` in (4147,4148,4149,4150,4154,4155,4156,52594,52595,52596,89879,90128,90279,90280);
+INSERT INTO `game_event_creature` (`eventEntry`,`guid`) VALUES
+(-25,4147), -- Rams out of Corral
+(-25,4148),
+(-25,4149),
+(-25,4150), -- Ultham Ironhorn out of Corral
+(-25,4154), -- More Rams out of Corral
+(-25,4155),
+(-25,4156),
+(-25,52594), -- Fleshrippers replace Ghouls in Westfall
+(-25,52595),
+(-25,52596),
+(-25,90279), 
+(-25,90280),
+(-25,89879), -- Defias Smugglers
+(-25,90128);
+
+-- -------------------------------------New Evening Event-------------------------------------------
+
+DELETE FROM `game_event` WHERE `eventEntry` = 88;
+INSERT INTO `game_event` (`eventEntry`,`start_time`,`end_time`,`occurence`,`length`,`holiday`,`holidayStage`,`description`,`world_event`,`announce`) VALUES
+(88,'2016-10-28 18:00:00','2030-12-30 23:00:00',1440,720,0,0,'Evening',0,2);
+
+-- Morning and Full Daytime Only Creatures/NPCs
+DELETE FROM `game_event_creature` WHERE `eventEntry` = -88 and `guid` in (86158,86159,86157,86354,86156,79700,79702,79815,79816,79817,87092,87088,87090,87089,87082,87091,84028,87023,79720,79721,86596,89294,86597,79803,79804,79812,90439,90440,90443);
+INSERT INTO `game_event_creature` (`eventEntry`,`guid`) VALUES
+(-88,79700), -- Adam and Billy
+(-88,79702),
+(-88,79815), -- Justin Brandon Roman
+(-88,79816),
+(-88,79817),
+(-88,87092), -- Miss Danna and students
+(-88,87088),
+(-88,87090),
+(-88,87089),
+(-88,87082),
+(-88,87091),
+(-88,84028),
+(-88,87023),
```

Comment:
![image](https://user-images.githubusercontent.com/16887899/160285299-e0bd2970-5fa9-4247-a8f0-f326bf5afd70.png)

Reply:
i will approve once the re-ran tests show green. thanks for the heads up that this needed the other pr.